apps/sync/test/unit/adapters/bookmarks_test.js

```diff
@@ -26,9 +26,8 @@ requireApp('sync/js/adapters/bookmarks.js');
 
 window.DataAdapters = {};
 
-suite('sync/adapters/bookmarks >', () => {
+suite('sync/adapters/bookmarks >', function() {
```

Comment:
Mocha breaks if you pass it lexically scoped functions, see https://github.com/mochajs/mocha/issues/1856#issuecomment-139129161. The tests will still run, but for instance setting `this.timeout(1000);` for a suite will have no effect, because the tests no longer run in the suite context.